appshared/src/main/java/io/reark/rxgithubapp/shared/utils/NullNetworkInstrumentation.java

```diff
@@ -26,16 +26,14 @@
 package io.reark.rxgithubapp.shared.utils;
 
 import android.support.annotation.NonNull;
-
-import com.squareup.okhttp.OkHttpClient;
-
 import io.reark.rxgithubapp.shared.network.NetworkInstrumentation;
+import okhttp3.OkHttpClient;
 
 public class NullNetworkInstrumentation implements NetworkInstrumentation<OkHttpClient> {
     @NonNull
     @Override
     public OkHttpClient decorateNetwork(@NonNull OkHttpClient httpClient) {
-        return httpClient;
+        return httpClient.newBuilder().build();
```

Comment:
The method is meant to decorate the httpClient instance given as argument. It's now changed to create (and decorate) a new client instead, which we shouldn't do unless we change the method to be "createAndDecorate" or something similar.

One unit test is failing because of this change, as it's trying to create a new instance of a mocked object. You can run the unit tests from Android Studio, or with `./gradlew test` on command line.

The API has changed so that we can't add instrumentation to OkHttpClient instances, so I think we should add them in the builder step instead. I created one option for this at https://github.com/reark/reark/commit/fa3e81367ada8417c60e00e807fb668377fe5ee3, could you amend your pull request with these changes? With these changes the NullNetworkInstrumentationTest had to be removed, as the Builder is a final class and can't thus be mocked.